components/gly_gen/utils.js

```diff
@@ -118,4 +118,57 @@ export function GlycanCrossRef({ crossref }) {
     </div>
   ) 
 
+}
+
+export async function glygenProteinSearchQuery(uniprot_canonical_accessions) {
+  // hit the /protein/search query endpoing to get the list id 
+  const search_query = `https://api.glygen.org/protein/search?query={%22operation%22:%22AND%22,%22query_type%22:%22search_protein%22,%22uniprot_canonical_ac%22:%22${uniprot_canonical_accessions.join(',')}%22}`
```

Comment:
Recommend using [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/URLSearchParams) here to deal with url encoding, of the parameter as well